web/client/components/plugins/PluginsContainer.jsx

```diff
@@ -118,9 +118,9 @@ class PluginsContainer extends React.Component {
         return plugins
             .filter((Plugin) => !PluginsUtils.handleExpression(this.getState, this.props.plugins && this.props.plugins.requires, Plugin.hide))
             .map(this.getPluginDescriptor)
-            .filter(plugin => PluginsUtils.filterDisabledPlugins({plugin: plugin && plugin.impl || plugin}, this.getState))
+            .filter(plugin => PluginsUtils.filterDisabledPlugins({plugin: plugin && plugin.impl || plugin, cfg: plugin && plugin.cfg || {}}, this.getState))
```

Comment:
Could you please insert a comment that explains what each filter does? 
Every time is very hard to understand the main purpose of these filtering